src/useFormState.ts

```diff
@@ -82,6 +83,7 @@ let pendingAutoSave = false;
  */
 export function useFormState<T, I>(opts: UseFormStateOpts<T, I>): ObjectState<T> {
```

Comment:
We also have a `useFormStates` (plural) that we'll need to handle this fo as well